### ISSUE TITLE
Drop EOL versions of MongoDB

### ISSUE THREAD

**Issue by cmars** [marcoceppi/layer-mongodb#2]
Versions of MongoDB earlier than 2.6 are EOL and unsupported. Even 2.6 is nearing EOL. We should not install these versions at all. On older series we should either:

a) Install from 10gen's repo, or
b) Install from a deb resource

**Comment by marcoceppi** [comment]
I agree, however I don't quite agree on the installing from outside. We should always present a sane offline way to deploy. We "support" mappings for older versions simply because the option to choose a version in config exists. The user can opt to move to 2.6 or beyond and get it from the "right" location.

I do agree we need a better story around this, how jarring would it be if we presented a deb as a resource for mongodb where it was always > 3.0? Is there a hard line for people on 2.6 to > 3.0?
